deploy/cloud/README.md

```diff
@@ -31,15 +31,19 @@ Create environment
    terraform apply --auto-approve -target "module.ec_deployment" -target "null_resource.rules" -target "null_resource.store_local_dashboard" -target "module.eks"
    ```
    (Note it may take more than 20 minutes to create all the resources)
-4. To create an agent policy and IAM role for EKS, run:
+4. To deploy nginx ingress controller, and ebs csi driver run:
+   ```bash
+   terraform apply --auto-approve -target "module.apps"
+   ``` 
+5. To create an agent policy and IAM role for EKS, run:
    ```bash
    terraform apply --auto-approve -target "module.api" -target "module.iam_eks_role"
    ```
-5. Finally, to deploy the agent on EKS, run:
+6. To deploy the agent on EKS run:
    ```bash
    terraform apply --auto-approve
    ```
-6. Run the following command to retrieve the access credentials for your EKS cluster and configure kubectl.
+7. Run the following command to retrieve the access credentials for your EKS cluster and configure kubectl.
```

Comment:
I think we should aspire for minimum number of steps.
Is there a real reason for having this additional step separate from other steps?

Reply:
Step 4 just explains how to deploy `module.apps` as separated module. Since `terraform apply --auto-approve` will deploy all modules it is not required to mention it here and I can remove this step. Currently this terraform project is only additional option deploy our product. Currently this terraform project is only additional option deploy our product. Since we still not decided if we want to use it in CI, it is less relevant.